fbw-common/src/systems/instruments/src/EFB/Navigation/Pages/LocalFilesPage/LocalFileChartUI.tsx

```diff
@@ -93,7 +93,7 @@ export const LocalFileChartUI = () => {
 
     try {
       // IMAGE or BOTH
-      if (selectedTabIndex === 0 || selectedTabIndex === 2) {
+      if (selectedTabType === 'IMAGE' || selectedTabType === 'BOTH') {
```

Comment:
Maybe the type of this selectedTabType should be an enum, I'll leave it to you though (same as above)

Reply:
Since we're gonna be rewriting this in v4 anyway, I just wanted to get it working. It's better now than a number at least :joy: